GalaxyZooTouchTable/GalaxyZooTouchTable/ViewModels/ClassificationPanelViewModel.cs

```diff
@@ -191,7 +192,12 @@ private void LoadCommands()
             OpenClassifier = new CustomCommand(OnOpenClassifier);
             SelectAnswer = new CustomCommand(OnSelectAnswer);
             ShowCloseConfirmation = new CustomCommand(OnShowCloseConfirmation);
-            SubmitClassification = new CustomCommand(OnSubmitClassification);
+            SubmitClassification = new CustomCommand(OnSubmitClassification, CanSubmitClassification);
```

Comment:
This is ultra minor and I don't think you have to change this, but it would make sense for the custom command to take parameters in the order it will use them (check if can submit classification, then act upon the submission). That would mean switching the order of the constructor params here.

Reply:
This does make sense. Unfortunately, I'd have to change the order of the arguments for all custom commands on the app, which is around 100 or so. This might be something worth doing a little further down the line when some of the current design issues are closed.